pkg/oci/load_balancer.go

```diff
@@ -267,7 +270,7 @@ func (cp *CloudProvider) createLoadBalancer(ctx context.Context, spec *LBSpec) (
 		Certificates:  certs,
 	}
 
-	glog.V(4).Infof("CreateLoadBalancerDetails: %#v", details.String())
+	logger.With("details", details).Debug("CreateLoadBalancerDetails")
```

Comment:
This is gonna be a HUGE log message.... What if the cluster has 500+ nodes? Heck even 20 nodes would be a lot for this haha.